spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe SessionsController, type: :controller do
+  before do
+    DatabaseCleaner.strategy = :transaction
+    OmniAuth.config.test_mode = true
+  end
+  describe "#create" do
+    before do
+      set_valid_omniauth
+      request.env["omniauth.auth"] = OmniAuth.config.mock_auth[:google_oauth2]
+    end
+    it "should successfully create a user" do
+      expect { post :create,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.